fury/tests/test_actors.py

```diff
@@ -1492,3 +1492,12 @@ def test_marker_actor(interactive=False):
     report = window.analyze_snapshot(arr, colors=colors)
     npt.assert_equal(report.objects, 12)
 
+
+def test_actors_primitives_count():
```

Comment:
First of all, thanks for taking the time to go through all those actors and adding support to the new functions. However, you are not testing if the proposed function works for all of them. Could you please improve this test by adding the rest of the actors you have modified?